frontend/src/components/Map/SearchMap/MapContainer.tsx

```diff
@@ -18,7 +18,7 @@ const MapContainer: React.FC<Props> = ({ children, whenCreated, hasZoomControl =
       center={mapConfig.searchMapCenter as [number, number]}
       zoom={mapConfig.searchMapZoom}
       maxZoom={mapConfig.maximumZoomLevel}
-      whenCreated={whenCreated}
+      ref={whenCreated}
```

Comment:
Yep, since Leaflet 1.9 (I'm not sure of the version), the `whenCreated` prop has been replaced by `whenReady` but it's not really the same callback function.  
When React-leaflet shipped version 4, the [release note](https://github.com/PaulLeCam/react-leaflet/releases/tag/v4.0.0) explicitly wrote:
> Removed `whenCreated` property from the `MapContainer` component (a `ref` callback can be used instead).